app/react/package.json

```diff
@@ -58,7 +58,7 @@
     "global": "^4.4.0",
     "lodash": "^4.17.20",
     "prop-types": "^15.7.2",
-    "react-dev-utils": "^11.0.3",
+    "react-dev-utils": "^11.0.4",
```

Comment:
💭 The `react-dev-utils` package doesn't have a version available yet which has a "safe" version of `browserslist` available. Does this change help?

Reply:
yes, you are right, I'm closing this PR.